testing/testing.go

```diff
@@ -186,13 +184,13 @@ go.exe run GO_SCRIPT_PATH %*
 	}
 }
 
-// RunGitServer starts serving git HTTP server and returns its address including port
-func RunGitServer(t *testing.T, gitRoot string) (hostPort string) {
-	l, err := net.Listen("tcp", "localhost:0")
+// RunGitServer starts serving git HTTP server and returns its address
+func RunGitServer(root string, t *testing.T) (url string) {
+	l, err := net.Listen("tcp", "127.0.0.1:0")
```

Comment:
using loopback interface on the very, very small change the environment does not have localhost dns set up.